toolkit/imageconfigs/additionalconfigs/cloud-init.cfg

```diff
@@ -54,12 +54,12 @@ cloud_config_modules:
 # this can be used by upstart jobs for 'start on cloud-config'.
  - ssh-import-id
  - set-passwords
- - package-update-upgrade-install
- - runcmd
  - yum-add-repo
+ - runcmd
 
 # The modules that run in the 'final' stage
 cloud_final_modules:
+ - package-update-upgrade-install
```

Comment:
Just curious, the upstream cloud.cfg.tmpl file you mentioned [link](https://github.com/canonical/cloud-init/blob/main/config/cloud.cfg.tmpl) uses `_` instead of `-` in the module name. Is this difference okay?

Reply:
Yes. "-" works same as "_", but "_" is encouraged to use by upstream now.